.github/workflows/php.yml

```diff
@@ -40,3 +46,7 @@ jobs:
       - name: PHP CodeSniffer
         if: success() || matrix.allow_failure
         run: phpcs -wps --colors
+
+      - name: PHPStan
+        uses: php-actions/phpstan@v3
+        if: always()
```

Comment:
Since we no longer use Psalm, please set this back to: `if: success() || matrix.allow_failure`.